openreview/venue_request/venue_request.py

```diff
@@ -73,6 +73,18 @@ def setup_bidding_stage(self):
             'bid_count': {
                 'description': 'Minimum bids one should make to mark bidding task completed for them. Default is 50.',
                 'value-regex': '[0-9]*'
+            },
+            'bid_options': {
+                'description': 'Comma separated values e.g. Very High, High, Neutral, Low, Very Low. Default values are Very High, High, Neutral, Low, Very Low',
+                'value-regex': '[\w\s,]+'
```

Comment:
one small details, can you add a default field here with the values `['Very High', 'High', 'Neutral', 'Low', 'Very Low']`

most of the PCs don't know which options to use, having a default value is helpful

Reply:
Sounds good! It's done!